docs/view_components/create_scenario.rst

```diff
@@ -0,0 +1,55 @@
+..
+    one can then cross link to this view component by using
+    :ref:`create_scenario-label`
+
+Create Scenario
+---------------
+
+Create scenario view-component enables the user to create a new scenario object where the energy system can be implemented.
+
+Attributes
+^^^^^^^^^^
+
+**Combination of text field and GUI elements**
+    Create Scenario view component can be a text field or a Graphical User Interface (GUI)
```

Comment:
A text field *is* a GUI. If you only have a text field, how can you create the scenario? Don't you need a few buttons?

Reply:
I am afraid that if I describe further, then it becomes the rendering I imagined